bigchaindb_driver/crypto.py

```diff
@@ -10,8 +10,11 @@
 CryptoKeypair = namedtuple('CryptoKeypair', ('private_key', 'public_key'))
 
 
-def generate_keypair():
+def generate_keypair(seed=None):
     """Generates a cryptographic key pair.
+    
+    Args:
+        seed (str) : Optional Seed, defaults to `None`
```

Comment:
This line of the docstring should be updated similar to the update suggested in https://github.com/bigchaindb/cryptoconditions/pull/116#discussion_r226828985